src/AppInstallerCLICore/Commands/COMCommand.cpp

```diff
@@ -21,7 +22,9 @@ namespace AppInstaller::CLI
             Workflow::ReportExecutionStage(ExecutionStage::Discovery) <<
             Workflow::SelectInstaller <<
             Workflow::EnsureApplicableInstaller <<
-            Workflow::DownloadSinglePackage;
+            Workflow::ReportIdentityAndInstallationDisclaimer <<
+            Workflow::ShowPromptsForSinglePackage(/* ensureAcceptance */ true) <<
+            Workflow::DownloadInstaller;
```

Comment:
Do you leave out Dependencies part on purpose? I don't see why it's necessary for this pr. But if you feel you need to, add a TODO here to add back dependency processing for COM (and we probabl;y need to fix this TODO before next preview release?)

Reply:
I added back dependencies and added a TODO statement. Will address this refactoring in a separate PR